hadoop-ozone/dist/src/main/compose/testlib.sh

```diff
@@ -72,21 +72,35 @@ start_docker_env(){
 ## @param        robot test file or directory relative to the smoketest dir
 execute_robot_test(){
   CONTAINER="$1"
-  TEST="$2"
+  shift 1 #Remove first argument which was the container name
+  ARGUMENTS=($@)
+  TEST="${ARGUMENTS[${#ARGUMENTS[@]}-1]}" #Use last element as the test name
+  unset 'ARGUMENTS[${#ARGUMENTS[@]}-1]' #Remove the last element, remainings are the custom parameters
```

Comment:
shellcheck:9: note: Expressions don't expand in single quotes, use double quotes for that. [SC2016]